src/init.c

```diff
@@ -48,6 +51,7 @@ _PG_fini(void)
 {
 	_process_utility_fini();
 	_planner_fini();
```

Comment:
Nope, and that would probably avoid some class of bugs. Will change in this PR and document.